2112.07523/2112.07523.yaml

```diff
@@ -93,71 +102,74 @@
       score: 
         Haining: 2
         Will: 
-        Yasaman: 
+        Yasaman: 2
+        # reasonable given no dispersion appears explicitly in excerpt
     parabolic|Dirac|cos:
       LLM: parabolic
       human: 
       score: 
         Haining: 2
         Will: 
-        Yasaman: 
+        Yasaman: 2
     electrons|holes:
       LLM: electrons and holes
       human: holes
       score: 
         Haining: 1
         Will: 
-        Yasaman: 
+        Yasaman: 0
+        # electron vs hole seems ambiguous from the excerpt (no mention of either individually); I deducted for not picking between the two.
```

Comment:
It is not stated in text, so I infer it from the sign of mass.
From Eq. (1) actually both particle and hole exist. So I would like to change it to full score.